### PR TITLE
Added ghc recipe

### PR DESCRIPTION
**Pull request by eggzilla**

### CODE REVIEW

**Comment by eggzilla** [comment]
@bgruening here we go again :-)

**Comment by bgruening** [comment]
@eggzilla I was able to get it to work see the commit history. Now I wanted to run a `make test` as part of the compilation step and that failed. Maybe you know if we can check more extensively the compilation.

**Comment by eggzilla** [comment]
@bgruening Awesome! I will investigate :call_me_sherlock:

**Comment by eggzilla** [comment]
Please do not merge yet, want to add more tests!

File: recipes/ghc/build.sh [hunk 1]
```diff
@@ -0,0 +1,12 @@
+#!/bin/bash
+
+export CFLAGS="-I$PREFIX/include" 
+export LDFLAGS="-L$PREFIX/lib"
+export LD_LIBRARY_PATH="$PREFIX/lib:$LD_LIBRARY_PATH"
+export LIBRARY_PATH="$PREFIX/lib:$LIBRARY_PATH"
```
Comment (jakirkham) [inline recipes/ghc/build.sh]: Any reason not to include the existing values for these environment variables?

**Comment by jakirkham** [comment]
Thanks for working on this, @eggzilla. Added a small comment above.

Should we add WIP to the PR title since this is still being worked on?

File: recipes/ghc/build.sh (hunk 1 above)
Comment (eggzilla) [reply recipes/ghc/build.sh]: Thanks for reviewing! There is no reason, just added the existing flag values :-)

**Comment by eggzilla** [comment]
Ok, thanks for waiting, I would recommend merging now :-)

**Comment by bgruening** [comment]
Awesome work!

**Pull request merged**